### ISSUE TITLE
what are scores mean?

### ISSUE THREAD

**Issue by AtefehBaniasadi** [yaojieliu/ECCV2020-STDN#18]
Hi

First of all, thank you for sharing your codes

I ran your code with a pre-trained model on my own dataset and for one of the images I got this score  "0.270,  -0.016,  -0.008,  -0.015,  -0.006", Could you please what these mean?

Thanks,

**Comment by harrismohammed** [comment]
[](url)
![98214042-8c945500-1f78-11eb-81c3-5ba111882d6c](https://user-images.githubusercontent.com/37656508/108817430-e8ddbc00-75dd-11eb-89ba-60a197634d74.png)


The Values mean M, s, b, C, T respectively.

![Screenshot from 2021-02-23 13-50-23](https://user-images.githubusercontent.com/37656508/108817599-23dfef80-75de-11eb-89de-24c6c4a4b4cf.png)

use this for G(I) 

I am not able to find Alpha0, If you find let me know 

![2](https://user-images.githubusercontent.com/37656508/108817732-5c7fc900-75de-11eb-92f3-a3217f195e8b.png)

**Comment by AtefehBaniasadi** [comment]
Hi

Thank you for your rapid response.

How can I use those scores to get the final decision? Indeed, I would like to know what is the label of each image (Real or Fake)? 

Which one of these values (M, s, b, C, T ) can help me know if the image is live or spoof

Many thanks,

**Comment by harrismohammed** [comment]
you need to use those M,s,b,C,T values in the formula i have attached above. Substitute M in equation 3 and use s,b,C,T in g(I) equation. 

Find the score by substituting the values in score equation

**Comment by AtefehBaniasadi** [comment]
Is there a python script in this repo to get this value(M,s,b,C,T) and compute the score?

**Comment by harrismohammed** [comment]
nope. it is not possible, you need to set alpha0 value by testing empirically, based on the dataset. you can use this code as a starter to train the dataset and test.

**Comment by santo4ul** [comment]
Calculating the final score seems a little complicated. 

How do we set alpha empirically? any thumb rule?

**Comment by Kapil-23** [comment]
Hi @santo4ul did you get clue to calculate score ?

**Comment by jin1041** [comment]
@Kapil-23 did you get clue to calculate score ?
